initializr-service/src/main/resources/application.yml

```diff
@@ -951,6 +958,67 @@ initializr:
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-ad-integration-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Cosmos DB (DocumentDB)
+          id: document-db
+          groupId: com.microsoft.azure
+          artifactId: azure-documentdb-spring-boot-starter
+          description: Spring Boot starter for Azure Document DB service
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-documentdb-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-documentdb-spring-boot-starter
+              description: Source code and README for the
+        - name: Azure Key Vault
+          id: key-vault
+          groupId: com.microsoft.azure
+          artifactId: azure-keyvault-secrets-spring-boot-starter
+          description: Spring value annotation integration with Azure Key Vault Secrets
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-keyvault-secrets-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-keyvault-secrets-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Storage
+          id: azure-storage
+          groupId: com.microsoft.azure
+          artifactId: azure-storage-spring-boot-starter
+          description: Spring Boot starter for Azure Storage service
+          links:
+            - rel: guide
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-samples/azure-storage-spring-boot-starter-sample
+              description: Sample code for using the starter
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-starters/azure-storage-spring-boot-starter
+              description: Source code and README for the starter
+        - name: Azure Support
+          id: azure-support
+          groupId: com.microsoft.azure
+          artifactId: azure-support
+          description: Auto-configuration for Azure services like Service Bus and Cloud Foundry support
+          links:
+            - rel: reference
+              href: https://github.com/Microsoft/azure-spring-boot/tree/master/azure-support
+              description: Source code and README
```

Comment:
I would like that we extend this a little bit. For a start a link to a page (or a section of a page) that explains how to use the starter would be nice. It could list everything that you support and provide references. 

What I envision is something like: "If you want to get started with storage, add this dependency, configure that property and we'll configure a bean of type X for you".

There is no need to replicate the information, you could simply list all the features with link to other readmes/sections.